site/src/examples/accordion/Group.tsx

```diff
@@ -1,41 +1,40 @@
 import { ReactElement } from "react";
 import {
   Accordion,
-  AccordionGroup as Group,
+  AccordionGroup,
```

Comment:
It's better if we can avoid renaming our components in examples, it might cause confusion